### DISCUSSION TITLE
Token is now missing

### DISCUSSION THREAD

**Discussion by myModelY** [bimmerconnected/bimmer_connected#682] [General]
Hi.
After update to v0.17 connection are good, but today no answer, because token is missing.
Why? what happens by bmw?
anybody help for me?
regards

**Comment by myModelY** [comment]
solved, thx
https://bimmer-connected.readthedocs.io/en/stable/captcha.html

**Comment by myModelY** [comment]
can you help me?
homeautomatication not allowed, but user pi allowed?
MyBMWCaptchaMissingError: Missing hCaptcha token for login. See https://bimmer-connected.readthedocs.io/en/stable/captcha.html
Client MyBMW sending CONNECT
Client MyBMW received CONNACK (0)
Client MyBMW sending DISCONNECT

**Reply by rikroe** [reply]
By default, the required file for CLI is stored in `~`, so your user's home directory.

Copy `.bimmerconnected.json` to your other user and it should work.
